### PR TITLE
Update documentation for local example after adding VTOrc to it

### PR DESCRIPTION
**Pull request by GuptaManan100**
## Description

The documentation for local-example has been updated in this PR after the changes made in https://github.com/vitessio/vitess/pull/11155

### CODE REVIEW

File: content/en/docs/15.0/get-started/local.md [hunk 1]
```diff
@@ -96,35 +96,128 @@ $ ./101_initial_cluster.sh
 add /vitess/global
 add /vitess/zone1
 add zone1 CellInfo
+Created cell: zone1
```
Comment (frouioui) [inline content/en/docs/15.0/get-started/local.md:99]: Unrelated to this change, but on line 91

`You should see output similar to the following:` -> `You should see an output similar to the following:`

Comment (frouioui) [reply content/en/docs/15.0/get-started/local.md:99]: Same thing in `content/en/docs/15.0/get-started/local-mac.md`

**Review by frouioui — APPROVED** [review]
Awesome!

File: content/en/docs/15.0/get-started/local.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/get-started/local.md:99]: Fixed.

**Pull request merged**